docs/team/cliff.md

```diff
@@ -5,6 +5,8 @@ title: John Doe's Project Portfolio Page
 
```

Comment:
Should the title (line 3) be your name instead of John Doe?

Reply:
will fix